libs/sdk-bindings/makefile

```diff
@@ -66,22 +65,30 @@ bindings-swift: ios-universal darwin-universal
 	rm bindings-swift/Sources/BreezSDK/breez_sdkFFI.modulemap
 
 kotlin: android
-	cargo run --features=uniffi/cli --bin uniffi-bindgen generate src/breez_sdk.udl --language kotlin -o ffi/kotlin 
+	cargo run --features=uniffi/cli --bin uniffi-bindgen generate src/breez_sdk.udl --language kotlin -o ffi/kotlin
 
 android: aarch64-linux-android armv7-linux-androideabi i686-linux-android x86_64-linux-android
 
 aarch64-linux-android: $(SOURCES) ndk-home
 	cargo ndk -t aarch64-linux-android -o ffi/kotlin/jniLibs build --release	
 
-
 armv7-linux-androideabi: $(SOURCES) ndk-home
 	cargo ndk -t armv7-linux-androideabi -o ffi/kotlin/jniLibs build --release	
 
 i686-linux-android: $(SOURCES) ndk-home
 	cargo ndk -t i686-linux-android -o ffi/kotlin/jniLibs build --release	
 
 x86_64-linux-android: $(SOURCES) ndk-home
-	cargo ndk -t x86_64-linux-android -o ffi/kotlin/jniLibs build --release	
+	cargo ndk -t x86_64-linux-android -o ffi/kotlin/jniLibs build --release
+
+bindings-android: android
+	cp -r ffi/kotlin/jniLibs bindings-android/lib/src/main
+	sed 's/package_name = "breez_sdk"/package_name = "technology.breez"/' uniffi.toml > uniffi.android.toml
```

Comment:
You're right. Originally, I was thinking that `technology.breez` was closer to the convention on Android / in the Java world but I didn't want to break the other integrations so I changed it just for Android. But thinking about it now, I believe consistency trumps convention. Will change this so `breez_sdk` is the package name on Android as well.

Reply:
Also @cnixbtc we want to change the RN package to be dependent on this aar file (instead of requiring developers to build the binaries themselves). And for that it must be the same package used in the RN bundle.

Reply:
Makes sense! I'm still looking into what seems to be a runtime linker issue (see comment further down). I'll update this PR with a fix for that and update the package name before we merge it.